packages/coil-anonymous-tokens/test/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   // FROM UPKEEP TEMPLATE
-  "extends": "../../../tsconfig.json"
+  "extends": "../../../tsconfig.json",
```

Comment:
want to leave the root tsconfig for the ide, with these types of options disabled (for performance reasons - *assuming* it actually is a perf hit)